### ISSUE TITLE
Porting AudibleInstruments to v1

### ISSUE THREAD

**Issue by senisioi** [VCVRack/AudibleInstruments#76]
I am porting the modules to version 1, are there any guidelines or things I should be careful about when doing this?

**Comment by AndrewBelt** [comment]
This is on my to-do list when the v1 API is stable, so don't worry about it. I expect to be 3-5x more efficient than others who are not familiar with the v1 API.

**Comment by senisioi** [comment]
Sounds good. I already did some work on some modules [here](https://github.com/senisioi/AudibleInstruments/tree/v0.6/src). I'm going ahead anyways because I like to use the MIDI mapping from v1 with some 0.6 patches.
Also, I noticed in the modules that there are lots of hardcoded values added for the multiplication of different params, e.g. in Elements there's, `3.3*dsp::quarticBipolar(params[FM_PARAM].getValue()) * 49.5` or `params[FINE_PARAM].getValue() + 69.0`
Did you use any source for multiplying these or is it just trial and error?
